practice/src/css/components/text.css

```diff
@@ -0,0 +1,34 @@
+.text {
+  color: var(--dark-text-color);
+  font-weight: var(--font-weight-semibold);
+  font-family: var(--primary-font); 
+}
+
+.subheading {
+  color: var(--secondary-text-color);
+  font-size: var(--font-size-xsmall);
+  line-height: 22.4px;
+}
+
+.title {
+  font-size: var(--font-size-2xl);
+  line-height: 50.4px;
+}
+
+.subtitle {
```

Comment:
The class name is not good. You can refer to  some examples below:
1. main-heading
2. secondary-heading
3. main-title
4. secondary-title